browser/scripts/abstractObjectGazePlugin.js

```diff
@@ -182,8 +182,10 @@
 
 		findSelectableMeshNodes()
 		.map(function(selectable) {
+			var nodeRef = selectable.node.parent_graph.uid + '.' + selectable.node.uid
```

Comment:
a plugin that knows about parent graphs.. is this something the node can return? looks reusable.